src/common/snippets/src/lowered/pass/mark_invariant_shape_path.cpp

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+
+#include "snippets/lowered/pass/mark_invariant_shape_path.hpp"
+
+#include "snippets/lowered/expressions/buffer_expression.hpp"
+#include "snippets/op/memory_access.hpp"
+#include "snippets/snippets_isa.hpp"
+#include "snippets/utils/utils.hpp"
+#include "snippets/itt.hpp"
+
+namespace ov {
+namespace snippets {
+namespace lowered {
+namespace pass {
+
+namespace {
+
+static size_t NOT_AFFECTING_PATH = SIZE_MAX;
```

Comment:
```suggestion
static const size_t NOT_AFFECTING_PATH = SIZE_MAX;
```
Could you please also add a comment describing for what cases this special value is needed?